bindata/tnfdeployment/authjob.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  labels:
+    app.kubernetes.io/name: tnf-auth
+  namespace: openshift-etcd
+  name: tnf-auth
+spec:
+  template:
+    metadata:
+      annotations:
+        openshift.io/required-scc: "privileged"
+    spec:
+      containers:
+        - name: tnf-auth
+          image: quay.io/openshift/origin-cluster-etcd-operator
+          imagePullPolicy: Always
+          command: [ "tnf-setup-runner", "auth" ]
+          resources:
+            requests:
+              cpu: 50m
+              memory: 64Mi
+            limits:
+              cpu: 500m
+              memory: 128Mi
+          securityContext:
+            privileged: true
+            allowPrivilegeEscalation: true
+      hostIPC: false
+      hostNetwork: false
+      hostPID: true
+      serviceAccountName: tnf-setup-manager
+      terminationGracePeriodSeconds: 10
+      restartPolicy: Never
+  backoffLimit: 3
```

Comment:
I don't know that we want a backoff limit for this job. I tested this in AWS, and it failed because of firewalls blocking pcs traffic, which was something I forgot about entirely.

Instead of a back-off, is there a way to rate-limit them so it only retrys once a minute or something?

Reply:
I think that's fair. But does something relaunch the job after? Because otherwise if you've failed once, you're never getting out of that state.